seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/dynamiccompile/parse/AbstractParse.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.transform.dynamiccompile.parse;
+
+import java.io.Serializable;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public abstract class AbstractParse implements Serializable {
+
+    public abstract Class<?> parseClass(String sourceCode);
+
+    public Object invoke(Class<?> clazz, String methodName, Object... args)
+            throws InstantiationException, IllegalAccessException, InvocationTargetException {
+
+        for (Method method : clazz.getMethods()) {
+            if (method.getName().equals(methodName)) {
+                return method.invoke(clazz.newInstance(), args);
+            }
+        }
```

Comment:
Maybe we should use https://github.com/apache/seatunnel/blob/d296842c9c27cb2b1ec5194dd68d2e900ece9d49/seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/ReflectionUtils.java#L91
when can not find method should throw exception.